release/release_tests.yaml

```diff
@@ -1258,38 +1255,19 @@
     wait_for_nodes:
       num_nodes: 4
 
+  variations:
+    - __suffix__: aws
+    - __suffix__: gce
```

Comment:
Can we just use `suffix` instead? I think it should be fine because we disallow this in the base definitions anyway. So we don't need to care about scope here

Reply:
I'm happy with either one, @aslonnie ?

Reply:
maybe `_suffix` if you want it to be shorter? I would like to have something that indicates that this is a special field, where it will be eaten during the process. It is nice that it has a check now; I still think it is better to look a bit different from a normal field for readability purpose.